Applications/Tizen_native/CustomShortcut/src/data.c

```diff
@@ -171,7 +183,7 @@ static int _run_nnpipeline(appdata_s *ad, const char *src, bool append) {
           "tensor_transform mode=arithmetic option=%s ! "
           "tensor_filter framework=tensorflow-lite model=%s ! "
           "tensor_sink name=sink",
-          src, "typecast:float32,add:-127.5,div:127.5", model_path);
+          src, "typecast:float32,add:-127.5,div:-127.5", model_path);
```

Comment:
I think ( in - 127.5 ) / 127.5 is right one.

Reply:
Because this image is written on black background with white ink. I multiplied with negative. Should I revert this change?

Reply:
black background with white ink is the normal scenario. 
(in - 127.5)/127.5 is better IMO.

Reply:
Fixed.